README.md

```diff
@@ -63,29 +63,31 @@ Individual fundraisers participating in an active Participant Event or Personal
 
 |Field|Description|
 |---|---|
-|```avatarImageURL```|The URL associated with the Participant|
-|```campaignDate```|(Personal Campaigns only) The date of the Personal Campaign|
-|```campaignName```|(Personal Campaigns only) The name of the Personal Campaign|
-|```createdDateUTC```|The ISO-8601-formatted date (in UTC) this Participant was created|
-|```displayName```|The Participant's name|
-|```eventID```|The ID of the Event this Participant is associated with|
-|```eventName```|The name of the Event this Participant is associated with|
-|```fundraisingGoal```|The Participant's individual goal|
-|```isTeamCaptain```|(Team Members only) If (true), this Participant is the captain of the team|
-|```numDonations```|The number of donations this Participant has received|
-|```participantID```|The unique ID of the Participant|
-|```sumDonations```|The total sum of donations this Participant has received|
-|```teamID```|(Team Participants only) The ID of the Team this Participant is associated with|
-|```teamName```|(Team Participants only) The name of the Team this Participant is associated with|
+|`avatarImageURL` (string)|The URL associated with the Participant|
```

Comment:
@timmixell do you think this description should make it clear that it's the URL for the image? I know it has "avatarImage" in the name, but it might help to make it explicit. (and we are being pretty explicit/idiot-proof in some of the others, like campaignName)